DrawerLayout.js

```diff
@@ -1,2 +1,15 @@
-import DrawerLayout from './src/components/DrawerLayout';
+/*
+
+This is a terrible hack to ensure that every user of RNGH can import Swipeable and DrawerLayout
+via scoped import like "import Swipeable from 'react-native-gesture-handler/Swipeable';".
+
+Even if you set main file to be in the dist/ directory it doesn't mean that scoped file resolution
+will be performed in that director. Actually node will search in <root>/Swipeable (which doesn't exist with .js extension)
+
+This behavior can be mitigated via "exports" key in package.json but it's supported since Node 14 which is quite new.
+
+Please remove this file when we stop supporting node versions lower than 14.
+
+*/
```

Comment:
Can it be done in a better way if we drop backward compatibility in this migration? Maybe it is worth posting a breaking change for the sake of less tech debt?

Reply:
We can't use scoped imports with TS, so possible solutions are:

- drop node <14 compat
- copy all files from `dist/` after transpiling to the root directory
- maybe try using symlinks? Not sure if this will work, IIRC metro doesn't support symlinks
- move Swipeable and Drawer as exports, dropping `react-native-gesture-handler/Swipeable` import

Reply:
The last one sounds like a fine solution, at least to me. But I might be missing something important there as well.

Reply:
I think we should leave it for now and if the next PR breaks types, then we can do some more breaking changes.

Reply:
Can't we do something like `react-native-screens/native-stack`? It is in the `native-stack` folder in `src` with its own `index.tsx` file. Or am I missing something too?

Reply:
Using `name.js` or `name/index.js` is equivalent. I'll look at screens config, maybe we can figure something out.

Reply:
Fixed in 314bb40.

I managed to work around this by creating `{DrawerLayout, Swipeable}/package.json` and specifying the entry point there. It doesn't really differ from using `Swipeable.js` linking to `dist/` dir as it was done before but I find it a little clearer. It also allowed removing `exports` section in the `package.json`.